README.md

```diff
@@ -1,45 +1,73 @@
+7. The scouts then login and repeatedly query the scout endpoint until a match is avaliable (ie. step 4 is done).
+8. The match is scouted, then the scouts submit their results with the submit/match endpoint.
 
-First, make sure to create a secrets folder containing passwords.ts in the root directory. Passwords should look like this:
+### /scout
+This is very simple, it just exists to poll the next robot from the scouting queue.
 
-```
-export const user_password = "password";
-export const admin_password = "adminpassword";
-```
+This request is made by a client every second or so while they're logged in. As soon as a match is released by the admin, the client will receive a valid response.
 
-Then, once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), start a development server:
+### /authed
+This endpoint is also very simple. A request to it is made by the client if the entered passphrase matches the correct passphrase. The client needs to check the passphrase, since it's a svelte store, which only the it had access to.
 
-```bash
-npm run dev
+### Submit
+This route contains two endpoints, match, pit, and photo. Each one of for a different type of submission. These endpoints are requested when the client wants to submit scouting data.
 
-# or start the server and open the app in a new browser tab
-npm run dev -- --open
-```
+#### /submit/match 
+This endpoints basically just takes in all the data from a match, including the cycle and defense times, and puts them in the database. [Request made from here](/src/lib/components/match-scout-components/Submit.svelte)
 
-## Building
+#### /submit/pit
+This endpoint takes in the event and team keys as well as the pit-scouting data and inserts it into the database. [Request made from here](/src/lib/components/pit-scout-components/Submit.svelte)
 
-To create a production version of your app:
+#### /submit/photo
+This endpoint is slightly more complex. Requests to this route must an array or object of photos, and a team and event key. First, the team and event keys are validated by checking if they both match their expected format (see the validateInput function at this endpoint's route). In the future, this could extend to also making sure that the given team is present at the event, using a request to TBA's API. Then each photo is validated by making sure they all less than one gigabyte in size. Finally, all the photos are inserted into the database. [Request made from here](/src/routes/photo/+page.svelte)
 
-```bash
-npm run build
-```
+### Admin
+This route is by far the most complex, and contains several endpoints, assign, authed, make-event, and teams. Each one is used for a different perpose by the admin, and should only be accessable by the admin. Each of these endpoints requires the admin password header to succeed. [All requests made from here](/src/routes/admin-dashboard/+page.svlete)
```

Comment:
accessable -> accessible
Instead, write "Each of these endpoints requires an admin password header to work." (maybe not "work" tho)